R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -35,6 +38,8 @@ TableHandle <- R6Class("TableHandle",
                 you are trying to call the constructor of TableHandle directly, which is not advised.")
             }
             private$internal_table_handle <- table_handle
+            private$is_static_field <- private$internal_table_handle$is_static()
+            private$num_rows_field <- private$internal_table_handle$num_rows()
```

Comment:
I think these variables are unnecessary, since it looks like the values are retrieved via function.